### ISSUE TITLE
Overlay another Image on video when one already exists

### ISSUE THREAD

**Issue by DanitiKain** [jliljebl/flowblade-forum#2]
This is best explained with a video that I have shared.

https://streamable.com/aymgj

**Comment by DanitiKain** [comment]
I was able to get it to look like this with several disolves added. However I can't get it to come to the front.

https://i.imgur.com/b1kUXfX.png

**Comment by DanitiKain** [comment]
https://jliljebl.github.io/flowblade/webhelp/compositor.html#Gimp/Photoshop_style_layer_order_yields_wrong_result

I think I found the guide here but it is not quite clicking a video posted on YT would help to show how this would work.

**Comment by jliljebl** [comment]
Yes, the top-to-bottom order is important. I can post a how-to-video sometime during this development cycle but here is a screenshot on how:
  * first fade in girl on white backgroud
  * fade text on top of both
![text_fade_on_top_girl](https://user-images.githubusercontent.com/1823444/52897196-61813500-31da-11e9-9d51-2bdedd2ff706.png)

Note the fade in keyframes on Dissolve Compositor just bringing the text in on playhead position.

**Comment by DanitiKain** [comment]
I finally figured this out thanks for the tips it helped.
